javascript/javascript1/week1/Ez-Namey.js

```diff
@@ -0,0 +1,30 @@
+const randomNumber = Math.floor(Math.random() * 10);
+let firstWords = [
+  "boss",
+  "pak",
+  "blast",
+  "sensor",
+  "loop",
+  "lethal",
+  "ace",
+  "techbia",
+  "chip",
+  "knight",
+];
+let secondWords = [
+  "smart",
+  "bold",
+  "super",
+  "fun",
+  "core",
+  "cool",
+  "Rentoor",
+  "Kiddily",
+  "Hiphonic",
+  "SalePush",
+];
+
+let startupName = firstWords[randomNumber] + secondWords[randomNumber];
+console.log(
+  `The startup:${startupName} contains ${startupName.length} characters.`
+);
```

Comment:
This can be a better way to put the first and second name together since you can add a space in between as well.
```suggestion
let startupName = `${firstWords[randomNumber]} ${secondWords[randomNumber]}` ;
console.log(
  `The startup:${startupName} contains ${startupName.length} characters.`
);
```